packages/components/src/presentation-components/instance-filter-builder/MultiTagSelect.tsx

```diff
@@ -86,12 +86,12 @@ function TagSelectOption<TOption, IsMulti extends boolean = boolean>({ children:
   const className = classnames("iui-menu-item", {
     "iui-focused": props.isFocused,
     "iui-active": props.isSelected,
-    "presentation-instance-filter-special-property-value": optionLabel === "",
+    "presentation-instance-filter-special-property-value": optionLabel === translate("unique-values-property-editor.empty-value"),
```

Comment:
It would be better to compare by option value to avoid localizing string just for comparison. Also for this to work you will need to update `getOptionValue` function that is passed to `AsyncMultiTagSelect` in `UniquePropertyValuesSelector.tsx`. I think it should be the same as `getOptionLabel` was before your changes.

Reply:
good call